frontend/src/components/common/podcast-item/podcast-item.tsx

```diff
@@ -8,23 +8,27 @@ type Props = {
 };
 
 const PodcastItem: React.FC<Props> = ({ podcast }) => (
-  <li className={styles.wrapper}>
-    <ImageWrapper
-      width="195"
-      height="195"
-      loading="lazy"
-      src={podcast.image?.url}
-      alt={podcast.name}
-      label={podcast.name}
-      className={styles.imageWrapper}
-    />
-    <h3 className={styles.title}>
-      <Link to={`${AppRoute.PODCASTS}/${podcast.id}`} className={styles.link}>
-        {podcast.name}
-      </Link>
-    </h3>
-    <p className={styles.author}>{podcast.user.nickname}</p>
-  </li>
+  <div>
```

Comment:
<li> should be a wrapper tag, because we use this component as list item

Reply:
I know, thats why I`m set this PR to draft